foundations/foundation-config/src/main/java/org/apache/servicecomb/config/ConfigUtil.java

```diff
@@ -171,7 +171,14 @@ private static void duplicateServiceCombConfigToCse(AbstractConfiguration source
       }
 
       String cseKey = CONFIG_CSE_PREFIX + key.substring(key.indexOf(".") + 1);
-      source.addProperty(cseKey, source.getProperty(key));
+      if (!source.containsKey(cseKey)) {
+        source.addProperty(cseKey, source.getProperty(key));
```

Comment:
source.setProperty is better?

Reply:
source.setProperty will clear and add. Here we checked existence and logged a warning message.

Reply:
ok